.github/workflows/publish-github-pages.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2021, Oracle and/or its affiliates.
```

Comment:
This file looks unrelated to the common mount feature - looks like an incorrect merge?

Reply:
Ditto for a number of the recent doc file changes.